screens/Menu/CourseScheduleDetails.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { View, Text, StyleSheet } from "react-native";
+import { Icon, Button } from "native-base";
+
+
+function CourseScheduleDetails(props) {
+
+    const goBack = () => {
+        props.navigation.goBack();
+    }
+
+    return (
+        <View style={styles.container}>
+            <View style={styles.menuButtonContainer}>
+                <Button transparent style={styles.menuButton} onPress={goBack}>
+                <Icon name="md-arrow-round-back" style={styles.icon} />
+                </Button>
+            </View>
+
+            <Text style={styles.mainLabel}>Class Details</Text>
+            
+            <Button transparent style={styles.routeButton} ><Text style={styles.viewRouteText}>View Route</Text></Button>
```

Comment:
Just to make sure, this button will take you to the Double Search screen with the _To_ field being the classroom?

Reply:
Yes, that would be the plan! The _classroom_ name from this screen would be the input value to feed the _To_ field in the _Double Search_ screen.